src/main/include/subsystems/Climber.h

```diff
@@ -0,0 +1,25 @@
+#include <rev/SparkMax.h>
+#include <frc2/command/SubsystemBase.h>
+#include <frc/controller/PIDController.h>
+#include "Constants.h"
+#include <frc2/command/CommandPtr.h>
+
+using namespace rev::spark;
+
+class Climber : public frc2::SubsystemBase {
+    public :
+
+    Climber();
+
+    frc2::CommandPtr FlipArm();
+
+    SparkMax m_motor;
+
+    private:
+
+    bool m_climber_up;
+
+    frc::PIDController m_pid_controller;
+
+
+};
```

Comment:
Indentation in this class is not consistent. Either the access modifiers should be at the same indentation as the `class` keyword (_my preference_):

```suggestion
class Climber : public frc2::SubsystemBase {
public:
    Climber();
    frc2::CommandPtr FlipArm();
    SparkMax m_motor;
private:
    bool m_climber_up;
    frc::PIDController m_pid_controller;
};
```

... or the access modifiers should be indented, _and_ the properties/methods should be additionally indented:

```suggestion
class Climber : public frc2::SubsystemBase {
    public :
        Climber();
        frc2::CommandPtr FlipArm();
        SparkMax m_motor;

    private:
        bool m_climber_up;
        frc::PIDController m_pid_controller;
};
```

Reply:
The Programming Conventions in the drive does not specify either way. But I agree with Mr. Collins that the access modifiers should be inline with the `class` keyword.

Reply:
We can update the Programming Conventions doc.